library/src/scala/quoted/Quotes.scala

```diff
@@ -55,7 +55,8 @@ trait Quotes { self: runtime.QuoteUnpickler & runtime.QuoteMatching =>
      *  Otherwise returns the value.
      */
     // TODO: deprecate in 3.1.0 and remove @experimental from valueOrAbort
-    // @deprecated("Use valueOrThrow", "3.1.0")
+    // TODO: deprecate in 3.2.0
```

Comment:
Why 3.2 ?

Reply:
I replaced the comments with find and replace. This one should not have been changed. Reverted.